src/main/java/com/werfen/report/service/GridReportService.java

```diff
@@ -18,20 +12,25 @@
 
 public class GridReportService {
 
+
     Logger log = Logger.getLogger(GridReportService.class.getName());
 
     public File build(GridReportConfiguration gridReportConfiguration, GridPageDataSource gridPageDataSource, ReportFormat reportFormat, PageFormat pageFormat) throws JRException {
 
         String filePath = gridReportConfiguration.getOutputFilePath() + reportFormat.getFileExtension();
         GridReportTemplateBuilder template = new GridReportTemplateBuilder();
-
         template.initJasperDesign("gridReport", pageFormat);
```

Comment:
Jasper design object is used only in PDF reports, so this part should be also moved to new PdfGridReportService. Moreover, to consider this done, this class should not know anything about jasper reports library (i.e shouldn't have imports)